de.tudarmstadt.ukp.wikipedia.api/pom.xml

```diff
@@ -59,21 +69,23 @@
 			<artifactId>nv-i18n</artifactId>
 		</dependency>
 
+        <!-- logger dependencies -->
 		<dependency>
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-api</artifactId>
 		</dependency>
 		<dependency>
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-log4j12</artifactId>
-			<scope>test</scope>
+			<scope>runtime</scope>
```

Comment:
`runtime` dependencies are transitive. Logging backend dependencies should not be included as transitive dependencies by libraries. Cf. https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope

Reply:
will fix